src/overlays/actors/ovl_En_Hata/z_en_hata.c

```diff
@@ -71,41 +83,48 @@ void EnHata_Update(Actor* thisx, GlobalContext* globalCtx2) {
     GlobalContext* globalCtx = globalCtx2;
     EnHata* this = THIS;
     s32 pitch;
-    Vec3f sp48 = sVec;
-    Vec3f sp3C;
+    Vec3f sZeroVec = { 0.0f, 0.0f, 0.0f };
+    Vec3f sWindVec;
```

Comment:
Ideally the `s` prefix scheme should be kept for static variables
```suggestion
    Vec3f zeroVec = { 0.0f, 0.0f, 0.0f };
    Vec3f windVec;
```